tasker/android.js

```diff
@@ -35,9 +35,11 @@ var getAndroidDevices = function () {
   if (res[0].indexOf("List of devices") == 0) {
     for (var i = 1; i < res.length; i++) {
       if (res[i].trim().length == 0) continue;
-      if (res[i].indexOf('offline') > 0) {
-        logme("Warning: Phone " + res[i] + " OFFLINE");
-        continue; // phone offline
+      if (res[i].indexOf('offline') > 0 ||
+          res[i].indexOf('unauthorized') > 0 ||
```

Comment:
This is a good point, but I tend to think `offline` and `unauthorized` are similar errors and should be treated similarly. They are similar because both means that a device can't be used, but do we want to fail the entire CI run due to that or try to log is visibly that a device needs attention?

My preference is to to keep the code as is now in the PR and just log the issue. This is because it might sometimes be that we need to get results from CI even when there isn't anyone who can manually fix an unauthorized issue on a single device.

On a side note, I have seen locally that sometimes `unauthorized` occurs even on a device that I have authorized and then doing something like `killall adb` might solve it (without authorizing anything from the device UI).